src/cmd.rs

```diff
@@ -398,10 +398,17 @@ fn initialize(root_path: String) -> Request<server::InitializeRequest> {
 }
 
 fn url(file_name: &str) -> Url {
-    let path = Path::new(file_name)
+    let canonical = Path::new(file_name)
         .canonicalize()
         .expect("Could not canonicalize file name");
-    Url::parse(&format!("file://{}", path.to_str().unwrap())).expect("Bad file name")
+    let mut path = canonical.to_str().unwrap();
+
+    // workaround for UNC path see https://github.com/rust-lang/rust/issues/42869
```

Comment:
@alexheretic I know this is a workaround, but do you think it might be a good idea to include a Windows-only test?

Reply:
I thought about that, but `"\\?\"` UNC just won't work wherever it's from so the windows check seemed redundant extra code. Can't see it coming from any other platform though, I think linking to the issue should allow removal if this ever gets solved in `canonicalize`

Reply:
I meant a Windows-only unit/regression test for the `url` func, so we'll know if/when that fails, but I'm not sure if it's even possible to create a reliable test for this use case in the AppVeyor CI :slightly_frowning_face: 

The comment alone is enough, just wanted to ask if it's a good idea to include such a test in the first place.

Reply:
I've added one now, left it running on all platforms as then all devs will keep it compiling (and it's a fast test).